web/src/upload.ts

```diff
@@ -424,9 +419,9 @@ export function setup_upload(config: Config): Uppy {
         // The uploaded files should be removed since uppy doesn't allow files in the store
         // to be re-uploaded again.
         uppy.removeFile(file.id);
-        // Hide upload status after waiting 100ms after the 1s transition to 100%
+        // Hide upload status after waiting 500ms after the 1s transition to 100%
         // so that the user can see the progress bar at 100%.
-        hide_upload_banner(uppy, config, file.id, 1100);
+        hide_upload_banner(uppy, config, file.id, 1500);
```

Comment:
I think the short 100ms time was intentional, not because of the send button thing fixed in the previous commit, which I think we'd not noticed, but just because it felt like it overstayed its welcome -- nobody needs to see a progress bar that's already complete for long, especially in the common case that the upload is almost immediate. Probably worth discussion in #design if we want to make this significantly higher.

Reply:
Hm, OK -- I'll drop the commit from the series.  This may just be a personal preference.